### ISSUE TITLE
How is the projectedScore calculate  for NBA Players

### ISSUE THREAD

**Issue by id021960** [sorare/api#638]
I noticed that the projections used by Sorare are consistently lower than those from SorareData and SorareNBAJet, even though all three platforms rely on Rotowire projections.

The official Sorare projections show him at only 39.5 points:
```json
{
  "data": {
    "players": {
      "displayName": "LaMelo Ball",
      "averageScore": 45,
      "anyPositions": [
        "basketball_guard"
      ],
      "age": 23,
      "country": {
        "name": "Vereinigte Staaten"
      },
      "anyFutureGames": {
        "nodes": [
          {
            "awayTeam": {
              "name": "Charlotte Hornets"
            },
            "date": "2024-11-24T01:00:00Z",
            "playerGameScore": {
              "projectedScore": 39.5
            }
          }
          
```
When I take the free projected stats for LaMelo Ball's next game from Rotowire:

![image](https://github.com/user-attachments/assets/bb0d5554-1bd3-43f6-87f7-372c5485cd04)

and calculate the score using the Sorare scoring matrix:

23.2 * 1 + 5.6 * 1.2 + 7.8 * 1.5 + 1.5 * 3 + 0.2 * 3 - 3.4 * 2 + 3 * 1

I get a projected score of 42.92, which is closer to SorareData's projection than Sorare's.

It seems that Sorare's projections are not calculated using the Sorare scoring matrix, or they apply some kind of adjustment that lowers the projected score.

Can someone explain how Sorare calculates projected scores?

**Comment by id021960** [comment]
If I leave out the additional points for three-pointers made, I would get a score close to the projected points from Sorare.

**Comment by elkesrio** [comment]
Please use the new field `projectedScore` defined in `ComposeTeamBenchObjectInterface` instead. 

A projected score has only sense in the context of a leaderboard. A game might be played in multiple fixtures (daily hoops and classic for example), hence different scoring matrices. 

Here is an example of you might use the new `projectedScore` field:

```graphql
query Test {
  node(id: "So5Leaderboard:<uuid>") {
    ... on So5Leaderboard {
      myBench(includeNoGame: true) {
        nodes {
          ... on ComposeTeamBenchObjectInterface {
            projectedScore(so5LeaderboardSlug: "<slug>")
          }
        }
      }
    }
  }
}
```

**Comment by id021960** [comment]
Thanks for the explanation! Unfortunately, the so5LeaderboardSlug I retrieved isn't working with the query you provided. Here's the query I used to obtain the so5LeaderboardSlug and id:

```graphql
query Leaderboards {
  so5 {
    so5Fixture(slug: "nba-26-28-nov-2024") {
      so5Leaderboard(so5LeaderboardType: NBA_LIMITED_CHAMPION) {
        id
        slug
      }
    }
  }
}
```
The id works perfectly, but when I use the slug in the projectedScore field, I don’t get the expected results. Is there a specific format or condition required for the so5LeaderboardSlug to function correctly in this context?

Additionally, is it possible to retrieve projections for all NBA players, not just the ones I own, especially for the daily competitions? This would greatly enhance lineup planning and strategy.

Thanks again for your help!
